docs/code-snippets/office-snippets.yaml

```
@@ -255,10 +255,21 @@ Office.AsyncResult#value:member:
     function write(message){
         document.getElementById('message').innerText += message; 
     }
+Office.Auth#getAccessToken:
```

This should cause the snippet to show up after being built:
```suggestion
Office.Auth#getAccessToken:member(1):
```